subgraphs/layer-one/.prettierignore

```diff
@@ -0,0 +1,2 @@
+build
+generated
```

Comment:
add new line at EOF

Reply:
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Reply:
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file